### DISCUSSION TITLE
Android Save Picture?

### DISCUSSION THREAD

**Discussion by navio-n** [unoplatform/uno#12078] [Q&A]
KnownFolders.PicturesLibrary is not implemented.

I can save the video to KnownFolders.VideosLibrary.

If I try to save an image to that folder error occurs. I can only save files with video file types to the video library.

**Comment by jeromelaban** [comment]
Thanks for the report. Could you provide some code which demonstrates the issue you're getting?

**Comment by navio-n** [comment]
Android 13:
	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />

Code that is working (KnownFolders.VideosLibrary and .mp4 file):

var storageFolderLocal = KnownFolders.VideosLibrary;
 _storageFileRecording = await storageFolderLocal.CreateFileAsync($"{DateTime.Now:yy-MM-dd-hh-mm-ss}.mp4", CreationCollisionOption.ReplaceExisting);

Code with error  (KnownFolders.VideosLibrary and .jpg file):
var storageFolderLocal = KnownFolders.VideosLibrary;
_storageFileRecording = await storageFolderLocal.CreateFileAsync($"{DateTime.Now:yy-MM-dd-hh-mm-ss}.jpg", CreationCollisionOption.ReplaceExisting);

ERROR: Access to the path "/storage/emulated/0/Movies/23-04-26-05-21-22.jpg" is denied.

If changed to KnownFolders.PicturesLibrary then the exception is:

The member StorageFolder KnownFolders.PicturesLibrary is not implemented. For more information, visit https://aka.platform.uno/notimplemented?m=StorageFolder%20KnownFolders.PicturesLibrary
